app/public/index.html

```diff
@@ -347,9 +348,29 @@
 					selele.append($('<option>').attr({title: SALSAH.userprofile.projects_info[p].longname}).val(p).text(SALSAH.userprofile.projects_info[p].shortname));
 				}
 			}
+		} else {
+			// user is not logged in,
+			// get all projects
+			limit_simple_search_get_all_projects();
 		}
 
-	};
+
+	}
+
+	/**
+	 * Gets all projects for selection in simple search field.
+	 */
+	function limit_simple_search_get_all_projects() {
```

Comment:
@lrosenth @subotic I could make this a function so it can also be called on logout.

Reply:
If it is not much work, then that would be great.

Reply:
sorry, this was misleading. I intended to say that I was able to do this :-)